services/workflows-service/prisma/schema.prisma

```diff
@@ -181,15 +185,15 @@ model Policy {
 }
 
 model Filter {
-  id String @id @default(cuid())
+  id        String   @id @default(cuid())
   // Name of the filter i.e. "KYC"
-  name String @unique
```

Comment:
Why was `@unique` removed?

Reply:
Not sure, maybe I missed it when I resolved conflicts. Thanks for catching it!